### ISSUE TITLE
Two wildcard parts in fields, second one not expanded ?

### ISSUE THREAD

**Issue by mircoc** [stalniy/casl#388]
**Describe the bug**
If the fields rule definition have more than one wildcard, only the the first one allow all values, the second one is considered literal.

**To Reproduce**
```js
const ability = new Ability([
    {
        action: "subscribe",
        subject: "Webhook",
        fields: [
            "vehicle.*.generic.*",
        ],
    }
]);

expect(
    ability.can("subscribe", "Webhook", "vehicle.*.generic.signal")
).toBe(true);
```

**Expected behavior**
The second wildcard should allow any string in that position.

**Interactive example** (optional, but highly desirable)
https://typescript-4anb33.stackblitz.io

**CASL Version**
`@casl/ability` 4.1.5

**Environment**:
"typescript": "~3.8.3"
node v12.16.2

**Comment by stalniy** [comment]
Hello, thanks for the issue.

This is because you pass a pattern :) The initial idea was that if you pass a pattern it will be an original pattern and I used `indexOf` for it. That's why i doesn't work

**Comment by stalniy** [comment]
Fixed  in @casl/ability@4.1.6
